src/taco-cli/cli/commands.json

```diff
@@ -265,19 +264,14 @@
                 "name": "search", "description": "[CommandPluginSearchSubcommandDescription]"
             }
             ],
-            "aliases": [
-            {
-                "alias": "plugins", "command": "plugin"
-            }
-            ,
-            {
-                "alias": "rm", "command": "remove"
-            }
-            ,
-            {
-                "alias": "ls", "command": "list"
+            "aliases": {
+                "plugins": "plugin",
+                "plugin": "plugin",
+                "remove": "remove",
+                "rm": "remove",
+                "list": "list",
+                "ls": "list"
```

Comment:
Should we include `"add": "add"` for completeness, even though it doesn't have any 'aliases' as such?